solutions/ipetek/turtlesim_mouse_tracker.cpp

```diff
@@ -0,0 +1,92 @@
+#include<ros/ros.h>
+#include<geometry_msgs/Twist.h>
+#include<geometry_msgs/Point.h>
+#include<turtlesim/Pose.h>
+#define _USE_MATH_DEFINES
+#include <cmath>
+#include <math.h>
+#include<stdio.h>
+
+
+int SCREEN_X = 1920; //EDIT
```

Comment:
Global variables like this would be nice to add as ROS parameters which can be run
http://wiki.ros.org/roscpp/Overview/Parameter%20Server

just by adding `__screen_x:=1920` as an argument and reading it through param server is usually done

Reply:
radi ako se seta sa rosparam set ... ali u argumentu zbog nekog razloga ne radi